### ISSUE TITLE
only get 57% acc on mv vector, ucf101 split1

### ISSUE THREAD

**Issue by DawnChou** [chaoyuaw/pytorch-coviar#89]
Hey, I just copied the options and settings when I trained coviar on UCF101-split1, but I only got 57% acc (validation) when training. Is there something wrong? Any help? Thanks.

**Comment by villawang** [comment]
Are you using the training script? Try testing the script. In the testing, they used more segments and also 10 crops as augmentation.

**Comment by JunLiangZ** [comment]
Hello, can you send me the "coviar" package, I think there is a problem with my installation, thank you, my email is 1789163036@qq.com

**Comment by Wendy-up** [comment]
> Are you using the training script? Try testing the script. In the testing, they used more segments and also 10 crops as augmentation.

Hey, I also followed the options and settings in this project on HMDB51, but got very low accuracy compared with the results in the paper, and I noticed that in the paper the author mentioned that their models were pretrained on ImageNet.
so dose this mean that I have to pretrain the model on ImageNet to get similar results with the paper, or just because there is something else wrong so that I get the poor results?
 Really thanks!
